src/php/tests/unit_tests/ChannelTest.php

```diff
@@ -37,6 +37,13 @@ public function testInsecureCredentials()
         $this->assertSame('Grpc\Channel', get_class($this->channel));
     }
 
+    public function testConstructCreatSsl()
```

Comment:
Actually, let's name this function `testConstructorCreateSsl` - there's a typo here anyways